Slingshot.Api/Concrete/BitbucketRepository.cs

```diff
@@ -234,6 +242,25 @@ public BitbucketRepository(Uri uri, string host, string token)
             }
         }
 
+        public override async Task<BitbucketRepository> GetPullRequest<BitbucketRepository>(string prId)
+        {
+            try
+            {
+                SourceControlInfo info = await this.GetScmInfo();
+                string repoToken = info.token;
+
+                using (HttpClient client = CreateHttpClient(accessToken: repoToken))
```

Comment:
We should think about a `CreateHttpClient` that takes care of token so we don't have to do the same thing each time.